client/app/bundles/course/assessment/components/AssessmentForm/translations.intl.js

```diff
@@ -137,58 +169,56 @@ const translations = defineMessages({
   },
   delayedGradePublication: {
     id: 'course.assessment.form.delayedGradePublication',
-    defaultMessage: 'Delayed Grade Publication',
+    defaultMessage: 'Enable delayed grade publication',
   },
   delayedGradePublicationHint: {
     id: 'course.assessment.form.delayedGradePublicationHint',
     defaultMessage:
-      "When delayed grade publication is enabled, gradings done by course staff will \
-      not be immediately shown to the student. To publish all gradings for this assessment, click \
-      on the 'Publish Grades' button on the top right of the submissions listing for this assessment.",
+      'When delayed grade publication is on, gradings will not be immediately shown to students. \
+      To publish all gradings, you may click Publish Grades in the Submissions page.',
   },
   showMcqMrqSolution: {
     id: 'course.assessment.form.showMcqMrqSolution',
-    defaultMessage: 'Show MCQ/MRQ Solution(s)',
-  },
-  showMcqMrqSolutionHint: {
```

Comment:
Nope, I intentionally removed it because it was no longer needed. The 'Show MCQ/MRQ solution(s)' checkbox is shown together with 'Show private test cases' and 'Show evaluation test cases', all three under the text 'After submission is graded and published'.